README.md

```diff
@@ -2,15 +2,12 @@
 
 Code repo for Delaware River Basin information theory code and machine learning models that predict estuary salinity
 
-This information is preliminary or provisional and is subject to revision. It is being provided to meet the need for timely best science. The information has not received final approval by the U.S. Geological Survey (USGS) and is provided on the condition that neither the USGS nor the U.S. Government shall be held liable for any damages resulting from the authorized or unauthorized use of the information.
-
 The steps for reproducing the results from the manuscript are detailed below. To retrain a new model use all steps. To analyze results from the manuscript run steps 1-3, then use steps 9-10 for analysis. Model ouptut used for the manuscript is located in `03_model/out/Manuscript_Results` 
 
 #### Steps for reproducibility:
 
 1) clone this repo using `git clone git@github.com:USGS-R/drb-estuary-salinity-ml.git --recurse-submodules`, the `--recurse-submodules` command initiates and updates the `river-dl` submodule housed in `03_model/src/` directory 
-2) add the file `953860.zip` into the `01_fetch/in` folder, the file can be found on S3 in drb_estuary_salinity/01_fetch/in
```

Comment:
We will want to include this input file in the model archive. We generally want to include the input data we used or at least use a DOI to link to the data source. The S3 storage is not guaranteed to exist long term. I went ahead and added the file to the repository because it is small. I also have some notes about how that data file was fetched that we might want to add to the metadata.xml file when @ted80810 is ready for it.